config/config_mgmt.py

```diff
@@ -60,17 +59,57 @@ def __init__(self, source="configDB", debug=False, allowExtraTables=True):
             else:
                 self.readConfigDBJson(source)
             # this will crop config, xlate and load.
-            self.sy.load_data(self.configdbJsonIn, self.allowExtraTables)
+            self.sy.load_data(self.configdbJsonIn)
```

Comment:
Why not calling "loadData" directly to reuse code?

Reply:
yes this can be called, since I am already calling other class functions in constructor.

Reply:
done.